src/components/workflowEditor/Workflow.tsx

```diff
@@ -278,6 +299,9 @@ export class Workflow extends Component<WorkflowProps, WorkflowState> {
                 }}
                 hideWebhookTippy={this.props.hideWebhookTippy}
                 addNewPipelineBlocked={this.props.addNewPipelineBlocked}
+                handleSelectedNodeChange={this.props.handleSelectedNodeChange}
+                selectedNode={this.props.selectedNode}
+                isLastNode={node.downstreams.length === 0}
```

Comment:
isLastNode is also available in API somewhere we should leverage that

Reply:
Since its correct for CDNodes only so using it there